torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -70,6 +75,16 @@ def forward(
         **kwargs,
     ) -> Explanation:
 
+        if isinstance(index, torch.Tensor) and (index.numel() > 1):
+            if not self._support_multiple_indices():
+                raise ValueError(
+                    f"{self.attribution_method.__name__} does not "
+                    "support multiple indices. Please use a single "
+                    "index or a different method.")
+            if self.kwargs.get('internal_batch_size', 1) != 1:
```

Comment:
We probably need to check if that argument is available for the given explainer. Shall we leave a TODO?

Reply:
Currently, we only support IG. When adding the support for more methods, I'll adjust those lines (added a reminder in the overall CaptumExplainer issue).